ocaml/idl/api_errors.ml

```diff
@@ -537,6 +537,7 @@ let suspend_image_not_accessible = "SUSPEND_IMAGE_NOT_ACCESSIBLE"
 (* PVS *)
 let pvs_site_contains_running_proxies = "PVS_SITE_CONTAINS_RUNNING_PROXIES"
 let pvs_site_contains_servers = "PVS_SITE_CONTAINS_SERVERS"
+let pvs_site_missing_servers = "PVS_SITE_MISSING_SERVERS"
```

Comment:
@robhoes has also pointed out to me that there shouldn't be a new API error defined as it gets caught immediately and is never displayed to a client; I will instead define just a normal exception for this part.